### PR TITLE
[Term Entry] /discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

-Added term **Proof by Cases** under concept _proofs_ within _discrete-math_.

Motivation:

The concept _proofs_ contains a term for one kind of proof technique within discrete math (namely, _proof by strong induction_), so I figured it would be helpful to have terms for other common proof techniques within discrete math.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (SSwiniarski) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:19]:
```suggestion
Proof by Cases is also sometimes known as **exhaustive proof**, which refers to the fact that the statement is proven in a set of mutually exhaustive cases.
```

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (SSwiniarski) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:34]: In the following example, can we rewrite so that we're not using the second-person pronoun, "we"?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@noahpgordon I did a first review and had a couple of comments.

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (noahpgordon) [reply content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:34]: I can do this, but perhaps the Docs convention should have an exception for math proofs? Use of 'we' is a standard convention in proofs. Let me know what you decide.

Comment (SSwiniarski) [reply content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:34]: @noahpgordon I'm inclined to make this exception, but I'll defer to the other maintainers: @yangc95, @KTom101, @caupolicandiaz, what do you think?

Comment (noahpgordon) [reply content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:34]: @SSwiniarski Shall we move forward with this?

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (KTom101) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:69]:
```suggestion
Therefore, proven that in either case, whether k is odd or even, 3k^2 + k is even. It follows that for any integer k, the product 3k^2 + k is even.
```

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (KTom101) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:37]:
```suggestion
Any integer is either odd or even. Therefore, consider two cases: (1) k is odd; and (2) k is even.
```

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (KTom101) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:61]:
```suggestion
Now, perform some algebraic manipulation on the expression 3k^2 + k:
```

File: content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md
Comment (KTom101) [inline content/discrete-math/concepts/proofs/terms/proof-by-cases/proof-by-cases.md:47]:
```suggestion
Now, perform some algebraic manipulation on the expression 3k^2 + k:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@noahpgordon To answer @SSwiniarski question, in mathematics, and by extension discrete mathematics, the use of "we" is considered a style decision rather than a strict rule. It all depends on the audience you're writing to. When writing always follow the style guide for who you're writing for.

In the case of Codecademy Docs entries, stick to our style guide and omit "we".

For example, line 47 can be changed from "We now perform some algebraic manipulation on the expression 3k^2 + k:" -> "Now, perform some algebraic manipulation on the expression 3k^2 + k:"

I commented these changes in the file.

**Comment by noahpgordon** [comment]
@KTom101 I've eliminated the use of 'we' in the proof example.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to me 🚀

**Comment by SSwiniarski** [comment]
@KTom101 we need your approval before we can merge this.

**Review by KTom101 — APPROVED** [review]
I made an additional direct change of converting the subheadings into AP title case.

**Pull request merged**
